pymatgen/util/io_utils.py

```diff
@@ -17,6 +17,29 @@
 __date__ = "Sep 23, 2011"
 
 
+def decompress_file_to_path(fin_path, fout_path=None):
```

Comment:
Isn't this quite similar to `monty.io.zopen`?

Reply:
Actually is more similar to `decompress_file`  in `monty` but the function won't return the path and delete the original compressed file. Created a helper function there as a fix

Reply:
I assume @shyuep wouldn't mind modifying `monty.shutil.decompress_file`, esp. since the doc string is currently wrong: arg `compression` doesn't actually exist.

https://github.com/materialsvirtuallab/monty/blob/2b91561d2483f2bebfc185c4945dfae85008c439/monty/shutil.py#L98

We can make a PR to monty to have the function return the path.

Reply:
@chiang-yuan Let's delete this function and use the changes in https://github.com/materialsvirtuallab/monty/pull/536 instead.

@shyuep Can we have a `monty` release whenever we do the next `pymatgen` release?